src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java

```diff
@@ -90,6 +97,13 @@ public Vocabulary importVocabulary(final boolean rename,
         }
         LOG.debug("Vocabulary import started.");
         parseDataFromStreams(mediaType, inputStreams);
+        LOG.debug("Checking that only language-tagged literals are provided.");
+
+        final Set<String> languageTags = getLanguageTagsPerProperties(model, multiLingualProperties);
+        if (languageTags.contains("")) {
+            throw new IllegalArgumentException("Each value of the properties must have a non-empty language tag: " + multiLingualProperties);
```

Comment:
Is the `IllegalArgumentException` handled gracefully in the REST exception handling? (Or caught somewhere and replaced with some general import exception)

Reply:
https://github.com/kbss-cvut/termit/blob/956ba990aab08790730b27f394bfa86e2e846bf6/src/main/java/cz/cvut/kbss/termit/service/repository/VocabularyRepositoryService.java#L188